kafe2/fit/histogram/cost.py

```diff
@@ -1,303 +1,21 @@
-from .._base import CostFunctionBase, CostFunctionBase_Chi2, CostFunctionBase_NegLogLikelihood, CostFunctionBase_NegLogLikelihoodRatio, CostFunctionException
```

Comment:
Why do we keep this file? It's unnecessary. Just for backwards compatibility?

Reply:
It's more for consistency regarding how kafe2 objects can be used.
For all other objects we have a Hist, Indexed, XY prefix.
So I imagine that a user would intuitively try to use a cost function with a prefix.
This also makes it possible to implement changes for only histogram/indexed should it become necessary.

Reply:
That's true. So we are keeping those Classes for now. Marking this as resolved.